src/platform/logging/index.ts

```diff
@@ -67,7 +67,7 @@ export function traceInfo(message: string, ...args: Arguments): void {
 }
 
 export function traceVerbose(message: string, ...args: Arguments): void {
-    if (globalLoggingLevel >= LogLevel.Debug) {
+    if (globalLoggingLevel <= LogLevel.Debug) {
```

Comment:
Submitting a PR now, as this increases the logging even in insiders, & when logging the telemetry its very noisy.
I think this was meant to be the other way around.
Basically if we want to log just errors today we will still log this information.

Reply:
The problem is it's not clear what the level means. Is higher stricter? It seems that way given this change, but maybe that's not clear everywhere else.